### DISCUSSION TITLE
How do I add a punctuation right after a linked message?

### DISCUSSION THREAD

**Discussion by NamesMT** [intlify/vue-i18n#1773] [Q&A]
Assume I have:
```
key_a: "Changed",
key_ab: "@:key_a : Something"
```
The following `key_ab` would output: "Changed : Something"
How do I move the punctuation to be right after `key_a`?, i.e: "Changed: Something"

**Comment by putianyi889** [comment]
See https://github.com/intlify/vue-i18n/issues/2163#issuecomment-3980357825
